src/components/ContentBlock.vue

```diff
@@ -1,7 +1,12 @@
 <template>
-  <section>
-    <h1>{{ nodeQuery.title }}</h1>
-    <div v-html="nodeQuery.body"></div>
+  <section :class="componentClass">
```

Comment:
We can have a base class and variants (`--something`), because some styles are reusable. We need to optimize and reuse as much CSS as possible.